Framework/DataHandling/test/LoadILLPolarizedDiffractionTest.h

```diff
@@ -0,0 +1,523 @@
+    TS_ASSERT_EQUALS(workspaceEntry1->y(1)[511], 0)
+    TS_ASSERT_DELTA(workspaceEntry1->e(1)[511], 0.00, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(130)[0], 180.00, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(130)[1], 186.64, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(130)[0], 0)
+    TS_ASSERT_DELTA(workspaceEntry1->e(130)[0], 0.00, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(130)[365], 2603.60, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(130)[366], 2610.24, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(130)[365], 0)
+    TS_ASSERT_DELTA(workspaceEntry1->e(130)[365], 0.00, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(131)[0], 180.00, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(131)[1], 186.64, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(131)[0], 0)
+    TS_ASSERT_DELTA(workspaceEntry1->e(131)[0], 0.00, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(131)[365], 2603.60, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(131)[366], 2610.24, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(131)[365], 1)
+    TS_ASSERT_DELTA(workspaceEntry1->e(131)[365], 1.00, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(132)[0], 180.00, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(132)[1], 186.64, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(132)[0], 5468)
+    TS_ASSERT_DELTA(workspaceEntry1->e(132)[0], 73.94, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(132)[511], 3573.04, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(132)[512], 3579.68, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(132)[511], 5394)
+    TS_ASSERT_DELTA(workspaceEntry1->e(132)[511], 73.44, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(133)[0], 180.00, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(133)[1], 186.64, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(133)[0], 0)
+    TS_ASSERT_DELTA(workspaceEntry1->e(133)[0], 0.00, 0.01)
+
+    TS_ASSERT_DELTA(workspaceEntry1->x(133)[511], 3573.04, 0.01)
+    TS_ASSERT_DELTA(workspaceEntry1->x(133)[512], 3579.68, 0.01)
+    TS_ASSERT_EQUALS(workspaceEntry1->y(133)[511], 0)
+    TS_ASSERT_DELTA(workspaceEntry1->e(133)[512], 0.00, 0.01)
+
+    MatrixWorkspace_sptr workspaceEntry2 =
+        std::dynamic_pointer_cast<Mantid::API::MatrixWorkspace>(
+            outputWS->getItem(1));
+    TS_ASSERT(workspaceEntry2)
+    TS_ASSERT_EQUALS(workspaceEntry2->getNumberHistograms(), 134)
+    TS_ASSERT_EQUALS(workspaceEntry2->blocksize(), 512)
+    TS_ASSERT(workspaceEntry2->detectorInfo().isMonitor(132))
+    TS_ASSERT(workspaceEntry2->detectorInfo().isMonitor(133))
+    TS_ASSERT(workspaceEntry2->isHistogramData())
+    TS_ASSERT(!workspaceEntry2->isDistribution())
+
+    TS_ASSERT_EQUALS(workspaceEntry2->getAxis(0)->unit()->caption(), "Time")
+    TS_ASSERT_EQUALS(workspaceEntry2->YUnitLabel(), "Counts")
+  }
+
+  void test_D7_multifile() {
+    // Tests 2 files for D7 with the generic Load on ADS
+    // This tests indirectly the confidence method
+    // (and NexusDescriptor issue therein)
```

Comment:
There is no issue here :)

Reply:
That's good to know, the comment was taken from ILLDiffractionTest and may not apply here.